AGPs/AGP-1H1.md

```diff
@@ -0,0 +1,44 @@
+---
+AGP: tbd
+Title: Conviction Voting + Template
+Author: @lkngtn (@1hive)
+Status: Stage II
+Track: Finance
+Created: 2019-11-12
+---
+
+# **AGP-TBD: Conviction Voting + Template**
+
+## **Address of the transfer recipient**
+
+ `0x41ecc492719a9cd750f8c9a693896d0dd621d4ec` Vault of the [conviction project organization](https://mainnet.aragon.org/#/conviction
+
+## **Amount of the transfer**
+
+75,000 DAI Due upon approval
+25,000 ANT Due upon successful delivery
+
+## **Number and frequency of transfers if recurring**
+
+2 (One upon approval, one upon successful delivery).
+For the purpose of budgeting, the full amount should be debited against this quarters budget.
+
+## **Purpose of the transfer**
+
+We believe that one of most significant blockers of wider adoption of Aragon is the lack of truly public, permissionless organizations. Organizations that anyone can join and participate in immediately without asking, while at the same time mitigating known deficiencies with participation and capture associated with fund allocation in public communities.
+
+In exchange for funding we will deliver the following as project deliverables:
```

Comment:
It is important to define in what state we expect to deliver our code. It is a little bit risky to promise production-ready code since conviction voting is still an experimental idea, so I think we can relax the expectation a little bit and promise well-crafted and user tested code. However, I think we are going to be able to overcome the possible problems and deliver the first version of Conviction Voting + Template ready to work for real DAOs.

Reply:
> It is a little bit risky to promise production-ready code since conviction voting is still an experimental idea, so I think we can relax the expectation a little bit and promise well-crafted and user tested code.

I think we should be able to provide some expectations around the utility of what we deliver, if we don't think we can deliver something that would be useful in production we should think about what it would take to get there. While conviction voting is still an experimental idea, it seems like it would be a useful experiment to be able to run.

Reply:
> it seems like it would be a useful experiment to be able to run

For sure, I can commit to that. We are going to run experiments as soon as we release a pre-version to rinkeby (mid-December), so I'm very confident that we'll be able to do enough development iterations informed by user's feedback to deliver useful code at the end of the funding.